### DISCUSSION TITLE
Chromium Standalone Extension

### DISCUSSION THREAD

**Discussion by EriahSylverstone** [aclap-dev/video-downloadhelper#919] [Help and Q&A]
### Your question

Would it be possible for the devs to just have a Chromium extension that isn't on the Chrome store, so the YouTube download function can be included on it. I'm currently using Opera GX, which is a Chromium browser, and was able to install the addon from the Chrome store, but without YouTube functionality, it's effectively useless to me. If Google doesn't allow the YouTube download, then perhaps just having a solo Chromium extension that's downloadable on the Video Downloadhelper website could be a workaround. And it can also make the functionality usable on Chrome, as well. It has to be possible to make the extension and not have to host it on Google's store where the restriction is in place. It also couldn't be too big of a file to host on their own website. People would just need to know how to install third party Chromium packages on their browser. And if they don't, instructions could be put up so they can.

**Comment by jcfrog** [comment]
Hi,
I'm not really sure we want to start a war against Google :)
